server/methods/deleteMessage.coffee

```diff
@@ -18,7 +17,7 @@ Meteor.methods
 
 		deleteQuery = 
 			_id: message._id
-		deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
+		#deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
```

Comment:
I get it, so the user could pass in { _id: original_message_id, u: { _id: hacker_user_id } } and they'd be deleting an arbitrary message. In this case, should we perform a find on rocketchat_message and validate that?